image_classification/tensorflow/official/utils/arg_parsers/parsers.py

```diff
@@ -160,6 +160,27 @@ def __init__(self, add_help=False, data_dir=True, model_dir=True,
           metavar="<BS>"
       )
 
+    if enable_lars:
+      self.add_argument(
+         "--enable_lars", "-el", type=Boolean, default=False,
```

Comment:
There is no type 'Boolean' defined. `action="store_true"` should do it.

Reply:
done